services/squash-api/values-squash-sandbox.yaml

```diff
@@ -0,0 +1,91 @@
+squash-api:
+  replicaCount: 1
+
+  image:
+    repository: lsstsqre/squash-restful-api
+    tag: latest
```

Comment:
This is a little scary in the long run since it means uploading a new Docker image will cause the API to be upgraded when it is restarted (which could occur for various reasons automatically and unpredictably in the middle of the night). I think it would be better in the long run to pin the version.

Reply:
Agreed. I didn't touch the SQuaSH API code base for a while but I'll soon change this.  In fact I'll take your previous suggestion and remove this from the yaml and use the default version set in the chart, which is `latest` right now but will change soon.